src/main.ts

```diff
@@ -322,6 +345,7 @@ export async function activate(
         return {
             initializationFinished: async () => {
                 await coreClrDebugPromise;
+                await razorLanguageServerStartedPromise;
```

Comment:
Switched Razor to activate down here so that C# DevKit can start initializing first